src/njoy21/input/ACER/Card5/Tempd.hpp

```diff
@@ -0,0 +1,13 @@
+struct Tempd {
```

Comment:
We should add this to `input::argument::common`. This card appears identical to previous `tempd`